packages/sui-mono/src/config.js

```diff
@@ -35,6 +36,15 @@ module.exports = {
   },
   getPublishAccess: function () {
     return publishAccess
+  },
+  getProjectName: function () {
+    return projectPackage.name
+  },
+  isMonoPackage: function () {
+    const folders = cwds(path.join(basePath, packagesFolder), deepLevel)
+    const scopePath = folders.pop()
+
```

Comment:
we can check if several of them have a package.json is a multi-package.

But:
- Should we check all of them?
- If one does not have the package.json is a multi-package? 
- If non of them but one have it, is multi-package?

Maybe we should check all of them and all should have the package.json.

Since the folders will be a scope and if you try to release that scope it will fail?